packages/design-tokens/package.json

```diff
@@ -54,6 +56,8 @@
     "@types/node": "^16.18.54",
     "clean-css-cli": "^5.6.3",
     "jest": "^29.7.0",
+    "postcss": "^8.4.47",
+    "react": "^18.2.0",
```

Comment:
Used for `getCSSVariablesFromStylesheet` in `packages/design-tokens/stories/test-utils/getCSSVariablesFromStylesheet.ts` and react for stories